loudness/templates/front-page.html

```diff
@@ -0,0 +1,14 @@
+<!-- wp:group {"style":{"spacing":{"blockGap":"0px"}},"layout":{"type":"default"}} -->
+<div class="wp-block-group">
+
+	<!-- wp:group {"backgroundColor":"primary","layout":{"type":"default"}} -->
+	<div class="wp-block-group has-primary-background-color has-background">
+		<!-- wp:template-part {"slug":"header","theme":"themes/loudness","tagName":"header"} /-->
+	</div>
+	<!-- /wp:group -->
+
+	<!-- wp:post-content {"tagName":"main","layout":{"inherit":true},"lock":{"move":false,"remove":true}} /-->
```

Comment:
I'm sorry! I realized after our conversation that this will not respect the Reading Settings. I.e. if the site is set to display latest posts, the home page will show the content of the most recent post. 

What do you think?

Reply:
... but if we include a query loop too?

Reply:
No... then it shows both.